src/tracing/mod.rs

```diff
@@ -173,6 +173,19 @@ impl TracingInspector {
         }
     }
 
+    /// Manually set the gas limit of the debug root trace.
```

Comment:
```suggestion
    /// Manually set the gas limit of the root trace.
```

Reply:
Hi @mattsse, the `debug` word here is to clarify this gas limit setting function only works for the Geth Debug trace root (to distinguish from the Parity trace root, which does not need to set this).